.github/workflows/pull-request.yaml

```diff
@@ -287,25 +282,20 @@ jobs:
         run: |
           kubectl port-forward \
             --namespace tracetest-integration \
-            svc/tracetest-integration 8080 &
+            svc/tracetest-integration 11633:8080 &
```

Comment:
You will have to update this right after the PR is merged, right?

Reply:
Yes. exactly. `integration` is still running on `8080`, so I need to talk to it, but leave it as ready as possible for later.
So this part is temporary until this PR is merged, and then I have to make a new PR to correctly point to integrations's `11633` port